src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2464,8 +2496,15 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
                   grainc_to_seed(p) = t1 * min(-cropseedc_deficit(p), grainc(p))
                   grainn_to_seed(p) = t1 * min(-cropseedn_deficit(p), grainn(p))
                   ! Send the remaining grain to the food product pool
-                  grainc_to_food(p) = t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
-                  grainn_to_food(p) = t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
+                  
+                  ! Y. Cheng, because I calculated grainc_to_food and grainn_to_food earlier
+                  if (ivt(p)==nmiscanthus .or. ivt(p)==nirrig_miscanthus .or. ivt(p)==nswitchgrass .or. ivt(p)==nirrig_switchgrass) then
+                  	grainc_to_food(p) = grainc_to_food(p) + t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
+                  	grainn_to_food(p) = grainn_to_food(p) + t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
+				  else
+				  	grainc_to_food(p) = t1 * grainc(p)  + cpool_to_grainc(p) - grainc_to_seed(p)
+                  	grainn_to_food(p) = t1 * grainn(p)  + npool_to_grainn(p) - grainn_to_seed(p)
+				  end if
```

Comment:
As you mentioned, you calculated grainc_to_food and grainn_to_food earlier, but doesn't look like you used the calculation based on the grainc and grainn pools (as in the original equations here). I'm new to looking at/reviewing code in GitHub, so sorry if I missed something.

Reply:
@danicalombardozzi  Lines 2505-2506 are the original calculation code. Because I calculated grainc_to_food(p) for switchgrass and miscanthus at line 2479 ahead, so I added line 2502-2503 just to add this previous calculated grainc_to_food(p) term (i.e., in line 2479).